### ISSUE TITLE
The best parameters to use during the training

### ISSUE THREAD

**Issue by TalapMukhamejan** [jumon/whisper-finetuning#29]
Hello, thanks for the repo. I am a newbie in terms of training the ASR model. I am currently fine-tuning this model for my language with over 1.5k hours of data. I assume in your case the parameters for that were different. I have been training with your default settings, then increased the number of steps to 10k and then to 30k. In the first two cases loss was >= 2, when I increased the number of steps to 30k, the loss gradually increased to 4 and I stopped the training. I want to decrease the number of steps but lower the learning rate to see what will happen. But I would appreciate it if you could help me set the right hyper-parameters for that.
